WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -196,6 +196,23 @@ final class OrderListViewModel {
         stores.dispatch(action)
     }
 
+    func dismissIPPFeedbackBanner(remindLater: Bool, remindAfter: Int?) {
```

Comment:
As mentioned before, `remindLater` and `remindAfter` shouldn't be exposed to the view, but kept here inside the view model logic.